src/app/configurator.cpp

```diff
@@ -104,15 +106,15 @@ namespace lean::app {
         ("base-path", po::value<std::string>(), "Set base path. All relative paths will be resolved based on this path.")
         ("data-dir", po::value<std::string>(), "Alias for \"--base-path\".")
         ("config,c", po::value<std::string>(),  "Optional. Filepath to load configuration from. Overrides default configuration values.")
-        ("genesis", po::value<std::string>(), "Set path to genesis config.yaml file.")
+        ("genesis", po::value<std::string>(), "Set path to genesis config yaml file (config.yaml).")
```

Comment:
How about to call it 'genesis.yaml'? 
'config.yaml' expected likely for `--config` arg.
```suggestion
        ("genesis", po::value<std::string>(), "Set path to genesis config yaml file (genesis.yaml).")
```